pwndbg/gdblib/disasm/__init__.py

```diff
@@ -386,6 +386,8 @@ def near(
         while insn is not None and len(insns) < instructions:
             if DEBUG_ENHANCEMENT:
                 print(f"Got instruction from cache, addr={cached:#x}")
+            if insn.jump_like and insn.split == SplitType.NO_SPLIT:
+                insn.split = SplitType.BRANCH_NOT_TAKEN
```

Comment:
Without this, branches in the disasm view that are before the current instruction pointer, but have all information cleared from the cache (possible in multiple scenarios) would have no split in the disasm view. This adds the split.